scripts/retract_registrations.py

```diff
@@ -0,0 +1,36 @@
+"""Script for retracting pending retractions that are more than 48 hours old."""
+
+import datetime
+import logging
+import sys
+
+from modularodm import Q
+
+from website import models, settings
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def main(dry_run=True):
+    pending_retractions = models.Retraction.find(Q('state', 'eq', 'pending'))
+    for retraction in pending_retractions:
+        if should_be_retracted(retraction):
+            retraction.state = 'retracted'
+            retraction.save()
+
+
+def should_be_retracted(retraction):
+    """Returns true if retraction was initiated more than 48 hours prior"""
+    return (datetime.datetime.utcnow() - retraction.initiation_date).days >= 2
```

Comment:
Might be useful to have the retraction pending time be configurable in website settings. It can be a timedelta value.

```python
# in defaults.py
RETRACTION_PENDING_TIME = datetime.timedelta(days=2)
```